code/index.html

```diff
@@ -22,7 +27,7 @@
     <nav class="navbar navbar-expand-md navbar-dark">
       <a href="../index.html">
         <div class="logosvg">
-          <img src="../common/img/logo.svg" />
+          <object data="../common/img/logo.svg"></object>
```

Comment:
The `object`  is not clickable, please add `pointer-events: none;` to its css definition.